docs/types/timestamp.md

```diff
@@ -2,12 +2,13 @@
 
 Parser operators can parse a timestamp and attach the resulting time value to a log entry.
 
-| Field         | Default    | Description                                                                   |
-| ---           | ---        | ---                                                                           |
-| `parse_from`  | required   | A [field](/docs/types/field.md) that indicates the field to be parsed as JSON |
-| `layout_type` | `strptime` | The type of timestamp. Valid values are `strptime`, `gotime`, and `epoch`     |
-| `layout`      | required   | The exact layout of the timestamp to be parsed                                |
-| `preserve`    | false      | Preserve the unparsed value on the record                                     |
```

Comment:
@djaglowski 

Does removing the `preserve` option break [this plugin](https://github.com/observIQ/stanza-plugins/blob/master/plugins/observiq_agent.yaml)?

I checked all the plugins, only the observiq_agent plugin is using `timestamp.preserve`.

Reply:
This is actually just a documentation issue that I noticed and fixed. this parameter was already removed in favor of `preserve_to`.